scripts/cxg_admin.py

```diff
@@ -76,6 +76,54 @@ def delete_dataset(ctx, uuid):
             exit(0)
 
 
+@cli.command()
+@click.argument("collection_name")
+@click.pass_context
+def delete_collections(ctx, collection_name):
+    """
+    Delete collections from data portal staging or dev by collection name.
+
+    You must first SSH into the target deployment using `make db/tunnel` before running.
+    You must first set DEPLOYMENT_STAGE as an env var before running
+    To run
+    ./scripts/cxg_admin.py --deployment dev delete-collections <collection_name>
+
+    Examples of valid collection_name:
+        - String with no spaces: ThisCollection
+        - String with spaces: "This Collection"
+    """
+
+    if ctx.obj['deployment'] == 'prod':
```

Comment:
Ah, in the  `click.group` above. 

Is this env always used in the url path, and not in the domain? Because if it gets used in the domain anywhere, then this would be vulnerable to accidental capitalizations, e.g. `Prod`